test/integration/relay/connection.test.js

```diff
@@ -193,12 +201,12 @@ if (helper.sequelize.dialect.name === 'postgres') {
           orderBy: new GraphQLEnumType({
             name: 'Viewer' + this.Task.name + 'ConnectionOrder',
             values: {
-              ID: {value: [this.Task.primaryKeyAttribute, 'ASC']},
+              ID: {value: [this.Task.primaryKeyAttribute, 'ASC']}
             }
           }),
-          before: (options, args, root) => {
+          before: (options, args, context, {viewer}) => {
             options.where = options.where || {};
-            options.where.userId = root.viewer.get('id');
+            options.where.userId = viewer.get('id');
             return options;
```

Comment:
Usually the test needs to be changed from `graphql(query, {values})` to `graphql(query, null, {values})`